### PR TITLE
Adding Nick Camerons talk at PDXRust 2016 to additional resources section

### PR DESCRIPTION
**Pull request by simonsan**
Thought might be helpful for people just looking at this repository but not knowing about the talk.

### CODE REVIEW

**Comment by simonsan** [comment]
Probably better in a `Materials` section to read more about some stuff rather than randomly stated in the doc. We could add the reddit thread from the #11 and some blog posts as well, plus the API guidelines.

@MarcoIeni What do you think?

**Comment by simonsan** [comment]
With the book already working we could probably add a small `materials` section or `further reading` (or something like this) in another md-file and include it in the TOC

**Comment by MarcoIeni** [comment]
> With the book already working we could probably add a small `materials` section or `further reading` (or something like this) in another md-file and include it in the TOC

sounds great!

**Comment by simonsan** [comment]
> 
> 
> > With the book already working we could probably add a small `materials` section or `further reading` (or something like this) in another md-file and include it in the TOC
> 
> sounds great!
> 
> Maybe `Additional materials`?
> 
> I don't like `further reading` because it will be a section with also videos, not only written stuff.

Japp, `Additional materials` sounds better to me as well

File: additional_materials.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Additional materials
+
+A collection of complementary helpful content
+
+## Talks
+
+- ["Design Patterns in Rust" by Nick Cameron at the PDRust (2016)](https://www.youtube.com/watch?v=Pm_oO0N5B9k)
```
Comment (MarcoIeni) [inline additional_materials.md:7]:
```suggestion
- [Design Patterns in Rust](https://www.youtube.com/watch?v=Pm_oO0N5B9k) by Nick Cameron at the PDRust (2016)
```

Comment (MarcoIeni) [reply additional_materials.md:7]: Do you think it's better like this? Anyway, feel free to keep this or your version and then you can merge it!

Comment (simonsan) [reply additional_materials.md]: I think I like that your version better. But you need to approve again after a change ;-)

**Comment by simonsan** [comment]
> 
> 
> > With the book already working we could probably add a small `materials` section or `further reading` (or something like this) in another md-file and include it in the TOC
> 
> sounds great!
> 
> Maybe `Additional materials`?

Hmm, thinking of it, what do you think of `Additional Resources`?

**Review by MarcoIeni — DISMISSED** [review]

**Review by MarcoIeni — APPROVED** [review]

**Pull request merged**